controllers/llamastackdistribution_controller.go

```diff
@@ -48,16 +50,19 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/filesys"
 )
 
+//go:embed all:manifests/base
+var manifests embed.FS
```

Comment:
❤️

Reply:
lol does this mean the conversation is resolved?